frontend/admin/src/js/components/user/UserTableFilterDialog.tsx

```diff
@@ -0,0 +1,284 @@
+import React, { useState } from "react";
+import Dialog from "@common/components/Dialog";
+import { Button } from "@common/components";
+import { FormattedMessage, useIntl } from "react-intl";
+import { BasicForm } from "@common/components/form";
+import SelectFieldV2 from "@common/components/form/Select/SelectFieldV2";
+import MultiSelectFieldV2 from "@common/components/form/MultiSelect/MultiSelectFieldV2";
+import "./UserTableFilterDialog.css";
+import { useFormContext } from "react-hook-form";
+import type { SubmitHandler } from "react-hook-form";
+import type { Option } from "@common/components/form/Select/SelectFieldV2";
+import { FilterIcon } from "@heroicons/react/outline";
+import useFilterOptions from "./useFilterOptions";
+import { ButtonIcon } from "../Table/tableComponents";
+
+export type FormValues = {
+  pools: Option["value"][];
+  languageAbility: Option["value"][];
+  classifications: Option["value"][];
+  operationalRequirement: Option["value"][];
+  workRegion: Option["value"][];
+  // TODO: Make mandatory once data model settles.
+  // See: https://www.figma.com/proto/XS4Ag6GWcgdq2dBlLzBkay?node-id=1064:5862#224617157
+  educationType?: Option["value"][];
+  employmentDuration: Option["value"][];
+  jobLookingStatus: Option["value"][];
+  skills: Option["value"][];
+  profileComplete: Option["value"][];
+  govEmployee: Option["value"][];
+};
+
+type FooterProps = Pick<UserTableFilterDialogProps, "enableEducationType">;
+const Footer = ({ enableEducationType }: FooterProps): JSX.Element => {
+  const { reset } = useFormContext();
+  const { emptyFormValues } = useFilterOptions(enableEducationType);
+  const handleClear = () => {
+    reset(emptyFormValues);
+  };
+
+  return (
+    <div style={{ display: "flex", placeContent: "space-between" }}>
+      <Button color="secondary" mode="outline" onClick={handleClear}>
+        <FormattedMessage
```

Comment:
By convention, we're not using `FormattedMessage` though I've long since forgotten why. :-)  Prefer the `formatMessage` functional form.